### DISCUSSION TITLE
D8F931B7-3567-3CF3-40AA-799F9ABAD720

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#115] [General]
# D8F931B7-3567-3CF3-40AA-799F9ABAD720

A catalog of 50 Luminous Blue Variable stars, including Eta Carinae.

https://celestia.mobi/addon?item=D8F931B7-3567-3CF3-40AA-799F9ABAD720

**Comment by exocritic5457** [comment]
The ascending node of either the HD 5980 AB or C barycenter is messed up. Also naming conflict with Eta Carinae.
